storagenode/peer.go

```diff
@@ -51,7 +51,7 @@ var (
 // architecture: Master Database
 type DB interface {
 	// CreateTables initializes the database
-	CreateTables() error
+	CreateTables(ctx context.Context) error
```

Comment:
This would end up inconsistent with what's in our documentation and satellite itself. Additional PR-s that fix `satellite.DB` and storj/docs repo would avoid this potential place of confusion -- i.e. why they are different.